packages/authorization-process/src/model/domain/errors.ts

```diff
@@ -93,14 +94,14 @@ export function purposeIdNotFound(
   });
 }
 
-export function securityUserNotFound(
-  requesterUserId: UserId,
-  userId: UserId
+export function userWithoutSecurityPrivileges(
+  userId: UserId,
+  requesterUserId: UserId
```

Comment:
about [this](https://github.com/pagopa/interop-be-monorepo/pull/571#discussion_r1639824641), my point was that the parameters should be
```suggestion
  consumerId: TenantId,
  requesterUserId: UserId
```

sorry for the previous unclear comment